socorro/external/es/crashstorage.py

```diff
@@ -273,10 +249,7 @@ def save_raw_and_processed(self, raw_crash, dumps, processed_crash, crash_id):
             'raw_crash': raw_crash
         }
 
-        self.transaction(
-            self._submit_crash_to_elasticsearch,
-            crash_document=crash_document
-        )
```

Comment:
Because `_submit_crash_to_elasticsearch` had its own retry logic, that meant we had DOUBLE RETRYING!